### PR TITLE
Fix favicon 404 and enhance to mobile support

### PR DESCRIPTION
**Pull request by landrok**
- Add an icon URL constant
- Add metas and fallback icons for old to modern browsers
- Add mobile icons for desktop shortcuts
- Update base template

### CODE REVIEW

**Comment by jpic** [comment]
Nice !

Instead of adding the STATIC_ICONS_URL setting, could we just use the `{% static %}` template tag from django ?

https://docs.djangoproject.com/en/1.10/howto/static-files/
http://blog.yourlabs.org/post/30382323418/surviving-djangocontribstaticfiles-or-how-to

File: templates/base.html [hunk 1]
```diff
@@ -2,10 +2,18 @@
 <html class="no-js" lang="en"> <!--<![endif]-->
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/png" href="{{ STATIC_URL }}img/favicon.ico" />
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ STATIC_ICONS_URL }}apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="{{ STATIC_ICONS_URL }}favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="{{ STATIC_ICONS_URL }}favicon-16x16.png" sizes="16x16">
```
Comment (jpic) [inline templates/base.html]: Could we use just `{% static 'favicon-16x16.png' %}` here and everywhere else and remove STATIC_ICONS_URL ?

**Comment by landrok** [comment]
That's ok, thanks for this advise, I'm (really) not a django expert.

Other thing, I saw there is a more recent branch, called "V3", maybe I should update this one too ?

**Comment by njoyard** [comment]
@landrok, thanks for the contribution. Indeed, you should work on v3 branch. This branch will disappear soon ;)

**Comment by landrok** [comment]
The branch has been removed, the patch is no longer relevant.

**Pull request closed without merging**